File: HD/hw22/sqrts.c


/* print square roots in C language.  R. Brown, 9/2010 */

#include <stdio.h>
#include <math.h>
#include <stdlib.h>

int main(int argc,char **argv)
{
  int n;
  int max;
  printf("sqrt(n)\n--------\n");
  max = atoi(argv[1]);
  for (n=0;  n<=max;  n++)
    printf("%f\n", sqrt(n));
  return 0;
}

File: HD/hw22/Sqrts.java
/* print square roots in Java language.  R. Brown, 9/2010 */

class Sqrts {
  public static void main(String args[]) {
    System.out.println("sqrt(n)");
    System.out.println("--------");
    int n;
    int max;
    max = Integer.parstInt(args[0])
    for (n=0;  n<=max;  n++)
      System.out.println(Math.sqrt(n));
    return;

  }
}
